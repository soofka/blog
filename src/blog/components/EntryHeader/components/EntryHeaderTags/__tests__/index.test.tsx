import { shallow } from 'enzyme';
import * as React from 'react';

import { EntryHeaderTags } from '../';

describe('<EntryHeaderTags/>', () => {

  it('renders correctly', () => {
    expect(shallow(
      <EntryHeaderTags
        tags={['tag1', 'tag2', 'tag3']}
      />,
    )).toMatchSnapshot();
  });

});
