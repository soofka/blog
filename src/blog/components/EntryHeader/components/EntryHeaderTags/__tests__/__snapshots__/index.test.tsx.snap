// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EntryHeaderTags/> renders correctly 1`] = `
<styled.p>
  <Link
    key="0"
    replace={false}
    to="/tag/tag1"
  >
    tag1
  </Link>
  <Link
    key="1"
    replace={false}
    to="/tag/tag2"
  >
    tag2
  </Link>
  <Link
    key="2"
    replace={false}
    to="/tag/tag3"
  >
    tag3
  </Link>
</styled.p>
`;
